### PR TITLE
Ocr function docker

### PR DESCRIPTION
**Pull request by ackaufman**

### CODE REVIEW

File: OcrFunctions/UploadOcrTrigger/__init__.py [hunk 1]
```diff
@@ -2,14 +2,32 @@
...
+
+    # TODO: Input validation, although what to do with bad input?
+    # Blob trigger functions don't support HTTP responses.
+
+    OcrService = AzureOcrService()
+    results = OcrService.get_ocr_results(ocr_service_url, subscription_key, blob_path)
+    output_text = OcrService.process_ocr_text(results)
+
+    outdata = {"ocr_text": output_text}
+    doc.set(func.Document.from_json(json.dumps(outdata)))
```
Comment (seushermsft) [inline OcrFunctions/UploadOcrTrigger/__init__.py]: Can you add the output format to the Gameplan document so we can agree on what it looks like?

**Comment by seushermsft** [comment]
run tests

**Comment by seushermsft** [comment]
run tests

**Comment by seushermsft** [comment]
It looks like there are some pylint failures. For W0511, it looks like you need to remove 'TODO' from line 225 in .pylintrc

You probably also need to add `relative-beyond-top-level` to line 144 of .pylintrc so it doesn't complain about the import statements.

File: OcrFunctions/UploadOcrTrigger/__init__.py [hunk 2]
```diff
@@ -2,14 +2,32 @@
...
+import json
 import azure.functions as func
+from ..shared.ocr_shared import AzureOcrService
+
+ocr_service_url = "https://westcentralus.api.cognitive.microsoft.com/vision/v2.0/ocr"
+blob_base_url = "https://boulderupskillstorage.blob.core.windows.net/"
+subscription_key = os.getenv('OCR_SUBSCRIPTION_KEY')
 
-def main(myblob: func.InputStream):
+def main(myblob: func.InputStream, doc: func.Out[func.Document]):
```
[the remark this answers is not in the document]
Comment (ackaufman) [reply OcrFunctions/UploadOcrTrigger/__init__.py]: Renamed myblob (default when you create a function from the template) to triggered_blob.

File: OcrFunctions/UploadOcrTrigger/__init__.py (hunk 1 above)
Comment (ackaufman) [reply OcrFunctions/UploadOcrTrigger/__init__.py]: Added to gameplan doc.

**Comment by seushermsft** [comment]
run tests

**Pull request merged**